cores/esp32/HardwareSerial.cpp

```diff
@@ -389,22 +398,17 @@ void HardwareSerial::updateBaudRate(unsigned long baud)
   uartSetBaudRate(_uart, baud);
 }
 
-void HardwareSerial::end(bool fullyTerminate)
+void HardwareSerial::end()
```

Comment:
This will break the UART CI test as it uses the fullyTerminate parameter

Reply:
The point is that `HardwareSerial::end(bool fullyTerminate)` is not necessary anymore, doesn't make sense with the changes and it is not Arduino HardwareSerial standard API.

Would it be possible to change the CI Test Code?
Or is it dependent from the API?

Reply:
@lucasssvaz - I changed the CI sketch. It is not breaking the UART CI anymore. Please review these changes.
I tested it here and all of them passed.